### DISCUSSION TITLE
Analog values on touchscreen

### DISCUSSION THREAD

**Discussion by pableitordasauto** [shanteacontrols/OpenDeck#297] [Ideas]
Hi Paradajz,
I see that the buttons for the Nextion touchscreen only accept digital states. Is it possible to adapt for analog values? The buttons accept continious values, so the touchscreen buttons can follow it as well. What do you think? Then, the touchscreen software can be configured with sliders to make more flexible the system.
For example, send values like:
_For slider 10:
release: printh 65 **(slider value)** 0A ff ff ff_

**Comment by pableitordasauto** [comment]
Not posible?

**Comment by paradajz** [comment]
That feature used to be supported but it didn't work quite reliably due to the lag from touchscreen so I've canned it.
